packages/serverless-logic-web-tools/src/i18n/locales/en.ts

```diff
@@ -62,6 +62,9 @@ export const en: AppI18n = {
       hostInfo: `The hostname associated with your instance.`,
       namespaceInfo: `The namespace (project) you want to deploy the model.`,
       tokenInfo: `The token associated with your instance.`,
+      insecurelyDisableTlsCertificateValidation: "Insecurely disable TLS certificate validation.",
```

Comment:
```suggestion
      insecurelyDisableTlsCertificateValidation: "Insecurely disable TLS certificate validation",
```
Removing the final period.